package.json

```diff
@@ -48,13 +48,17 @@
     "jshint": "jshint --show-non-errors ."
   },
   "dependencies": {
-    "sax": "~1.2.1",
     "coa": "~1.0.1",
-    "js-yaml": "~3.7.0",
     "colors": "~1.1.2",
-    "whet.extend": "~0.9.9",
+    "css-select": "^1.3.0-rc0",
+    "css-select-base-adapter": "^0.1.0",
+    "csso": "~2.3.1",
```

Comment:
CSSO 3.0 has been changed a lot, moved to use `css-tree` and had changes in AST format and API. I think SVGO should migrate to 3.0 first (I suppose no changes in sources is needed beside a version change in `package.json`).
New path for `specificity` is `/lib/restructure/prepare/specificity.js`. But that's unsafe to use it that way.
Anyway, feel free if you need help in migration to new AST format and API.